documentation/synthetic_tokens/tutorials/prerequisites.md

```diff
@@ -24,7 +24,8 @@ If everything worked, we should see the line "> Compiled successfully using:" in
 
 1. Install the [Ganache UI](https://truffleframework.com/ganache).
 2. Run Ganache on localhost port `9545` (use the above links for instructions on how to do this).
-3. Note that to deploy new `ExpiringMultiPartyCreator`, the main factory used in the priceless financial contract, you must set your gas limit to at least 9 million in Ganache. To do this click the gear icon at the top right of Ganache, then click "CHAIN" and set the "GAS LIMIT" field to `10000000`. Then click the "RESTART" button top right. This will enable your Ganache instance to deploy larger contracts.
+3. Under "Accounts & Keys", set the "Account Default Balance" to `100000`. This will ensure that you have enough local testnet ETH to run through tutorials.
+4. Note that to deploy new `ExpiringMultiPartyCreator`, the main factory used in the priceless financial contract, you must set your gas limit to at least 9 million in Ganache. To do this click the gear icon at the top right of Ganache, then click "CHAIN" and set the "GAS LIMIT" field to `10000000`. Then click the "RESTART" button top right. This will enable your Ganache instance to deploy larger contracts.
```

Comment:
Typo, should be `9000000`

Reply:
It might be helpful to include the CLI script in case users prefer that method: `ganache-cli -p 9545 -e 10000000000 -l 9000000`

Reply:
Thanks for flagging. I believe @chrismaree wrote this section initially, and I don't think it was a typo, since the instructions set the gas limit to 10mm, a bit higher than the minimum of 9 million. 

We can include the CLI script as well. I'll modify the amount of ETH to match the one in the words as well.

Reply:
Ah I see. It's a bit confusing to say set it to at least 9 million and then it sets it to 10million. Maybe just say you're gonna set it to 10 million

Reply:
ye, this is right. I just wanted to make the overhead a bit higher than where we are now so we can accommodate a bit of growth if we expand in the next bit. 10million is a save upper bound.